### ISSUE TITLE
First two controllers not working properly/at all

### ISSUE THREAD

**Issue by AdeizaSama** [Jays2Kings/DS4Windows#165]
Hi,

Thanks a lot for your software, it's been a big help. Recently though, I've had this weird issue. Assume my wireless devices are called "Wireless Controller" in the set up USB game controllers menu. Whenever I run DS4Windows and start, it used to change all that to "XBOX 360 Controller", as intended. All of them. Now, when I run it, it makes the first controller disappear completely, the second controller to be called "Wireless Controller" (recognised as a different device from the one excluded when I set to exclusive mode) and then finally the third controller as "XBOX 360 Controller".

To recap, P1 - Not working, P2 - Not working well, P3 - Works. It doesn't matter what order I sync the actual devices, this is what happens. The only one that will work is the third controller to be added. The profiles seem to work and so do the touch pads for all of them. All the buttons also work, but it just doesn't recognise the first 2 devices as an XBOX pad.

Any ideas on this? Thanks!
